src/Components/Table/Rows.js

```diff
@@ -2,26 +2,26 @@ import React from "react";
 import moment from "moment";
 
 const Rows = props => {
```

Comment:
You've named this component `Rows` (plural), but does it render 1 row or multiple rows? What do you think would be a better name?

Reply:
I think it will be Row